### ISSUE TITLE
Calculate enchanting level for enchantment tables.

### ISSUE THREAD

**Issue by skillman623** [Lea-fish/Leafish#271]
Enchantment tables should calculate max enchant based off of surrounding bookshelves.
There should be an option to support custom bookshelves too (such as Botania's mana pylon).

**Comment by cooltexture1** [comment]
Most of that logic is handled by the server tho. The server sends information about the level requirement and the possible enchantments via[ this packet](https://wiki.vg/Protocol#Set_Container_Property). In PR  #269 the enchantment table works by just displaying the information the server is sending.

**Comment by terrarier2111** [comment]
> Most of that logic is handled by the server tho. The server sends information about the level requirement and the possible enchantments via[ this packet](https://wiki.vg/Protocol#Set_Container_Property). In PR #269 the enchantment table works by just displaying the information the server is sending.

Do you think we can mark this as closed?

**Comment by cooltexture1** [comment]
yes i think so, although the enchatning table has an issue where ghost items often appear when using it. but the enchanting process itself and the display of the required levels works.
